src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs

```diff
@@ -530,5 +530,57 @@ public static string Page(this IUrlHelper urlHelper, string pageName, object val
                 host: host,
                 fragment: fragment);
         }
+
+        /// <summary>
+        /// Generates an absolute URL for a page, which contains the specified
+        /// <paramref name="pageName"/>, <paramref name="pageHandler"/>, route <paramref name="values"/>,
+        /// <paramref name="protocol"/> to use, <paramref name="host"/> name, and <paramref name="fragment"/>.
+        /// Generates an absolute URL if the <paramref name="protocol"/> and <paramref name="host"/> are
+        /// non-<c>null</c>. See the remarks section for important security information.
+        /// </summary>
+        /// <param name="urlHelper">The <see cref="IUrlHelper"/>.</param>
+        /// <param name="pageName">The page name to generate the url for. When not defined, defaults to the current executing page.</param>
```

Comment:
As in the #6775, this should be `When <see langword="null"/>`

Reply:
Huh, that should have been included in the last push, I already commited that :thinking: